projects/gloo/api/v1/enterprise/options/graphql/v1beta1/graphql.proto

```diff
@@ -13,131 +13,517 @@ option (extproto.equal_all) = true;
+  with extra templating capabilities. The following rules must be followed when using a templated strings:
+  - templated values will be interpolated into the strings they are in
+    * e.g. if `$parent.token` is `ABDEFG`, "Bearer_{$parent.token}" may become "Bearer_ABCDEFG"
+  - templated values must start with `{$`, immediately preceded by
+    - `parent` for extracting a value from the parent object
+    - `args` for extracting a value from the graphql field arguments
+    - `headers` for extracting a value from the request headers.
+  - `headers` extractions can only have one value in the path to represent the header being extracted
+  - for other extractions, e.g. `parent` and `args`, a more complicated path can be used for extracting nested values
+    using the following rules:
+      - use `a.b` to extract key "b" from object "a"
+      - use `a[0]` to extract the 1st element in array "a"
+      - use `a[*].b` to extract key "b" from all objects in array "a"
+      - use `a[*]` to create a list of key-value pairs from object "a"
+      - use `a[*][0]` to create a list of all keys from object "a"
+      - use `a[*][1]` to create a list of all values from object "a"
+ */
+
+// Deprecated: The GraphQL feature of Gloo Gateway will be removed in a future release.
+// Defines a configuration for generating outgoing requests for a resolver.
 message RequestTemplate {
-  reserved 1, 2, 3;
-  reserved "headers", "query_params", "body";
+  // Use this attribute to set request headers to your REST service. It consists of a
+  // map of strings to templated value strings. The string key determines the name of the
+  // resulting header, the value provided will be the value.
+  //
+  // The least needed here is the ":method" and ":path" headers.
+  // for example, if a header is an authorization token, taken from the graphql args,
+  // we can use the following configuration:
+  // headers:
+  //   Authorization: "Bearer {$args.token}"
+  map<string, string> headers = 1;
+
+  // Use this attribute to set query parameters to your REST service. It consists of a
+  // map of strings to templated value strings. The string key determines the name of the
+  // query param, the provided value will be the value. This value is appended to any
+  // value set to the :path header in `headers`.
+  //
+  // for example, if a query parameter is an id, taken from the graphql parent object,
+  // we can use the following configuration:
+  // queryParams:
+  //   id: "{$parent.id}"
+  map<string, string> query_params = 2;
+
+  // Used to construct the outgoing body to the upstream from the
+  // graphql value providers.
+  // All string values can be templated strings.
+  google.protobuf.Value body = 3;
 }
 
-// DEPRECATED: This message is deprecated and has been removed from use as of gloo 1.20. Message is being kept to prevent future use of these names and fields
+// Deprecated: The GraphQL feature of Gloo Gateway will be removed in a future release.
```

Comment:
Can we restore DEPRECATED these messages to indicate that they have been deprecated as of 1.20?